data/mods/PKs_Rebalance/pk_overmap.json

```diff
@@ -3,33 +3,33 @@
         "type" : "overmap_special",
         "id" : "Prison",
         "overmaps" : [
-            { "point":[-1,0,0], "overmap": "prison_3"},
-            { "point":[0,0,0], "overmap": "prison_2"},
-            { "point":[1,0,0], "overmap": "prison_1"},
-            { "point":[-1,1,0], "overmap": "prison_6"},
-            { "point":[0,1,0], "overmap": "prison_5"},
-            { "point":[1,1,0], "overmap": "prison_4"},
-            { "point":[-1,2,0], "overmap": "prison_9"},
-            { "point":[0,2,0], "overmap": "prison_8"},
-            { "point":[1,2,0], "overmap": "prison_7"},
-            { "point":[-1,0,-1], "overmap": "prison_b"},
-            { "point":[0,0,-1], "overmap": "prison_b_entrance"},
-            { "point":[1,0,-1], "overmap": "prison_b"},
-            { "point":[-1,1,-1], "overmap": "prison_b"},
-            { "point":[0,1,-1], "overmap": "prison_b"},
-            { "point":[1,1,-1], "overmap": "prison_b"},
-            { "point":[-1,2,-1], "overmap": "prison_b"},
-            { "point":[0,2,-1], "overmap": "prison_b"},
-            { "point":[1,2,-1], "overmap": "prison_b"}
+            { "point":[0,-1,0], "overmap": "road_end_north"},
+            { "point":[-1,0,0], "overmap": "prison_3_north"},
+            { "point":[0,0,0], "overmap": "prison_2_north"},
+            { "point":[1,0,0], "overmap": "prison_1_north"},
+            { "point":[-1,1,0], "overmap": "prison_6_north"},
+            { "point":[0,1,0], "overmap": "prison_5_north"},
+            { "point":[1,1,0], "overmap": "prison_4_north"},
+            { "point":[-1,2,0], "overmap": "prison_9_north"},
+            { "point":[0,2,0], "overmap": "prison_8_north"},
+            { "point":[1,2,0], "overmap": "prison_7_north"},
+            { "point":[-1,0,-1], "overmap": "prison_b_3_north"},
+            { "point":[0,0,-1], "overmap": "prison_b_2_north"},
+            { "point":[1,0,-1], "overmap": "prison_b_1_north"},
+            { "point":[-1,1,-1], "overmap": "prison_b_6_north"},
+            { "point":[0,1,-1], "overmap": "prison_b_5_north"},
+            { "point":[1,1,-1], "overmap": "prison_b_4_north"},
+            { "point":[-1,2,-1], "overmap": "prison_b_9_north"},
+            { "point":[0,2,-1], "overmap": "prison_b_8_north"},
+            { "point":[1,2,-1], "overmap": "prison_b_7_north"}
         ],
         "connections" : [
-            { "point" : [0,-1,0], "terrain" : "road" }
+            { "point" : [0,-1,0] }
         ],
-        "locations" : [ "land" ],
+        "locations" : [ "land", "swamp" ],
```

Comment:
You want prisons to generate in the swamp without road connections? That leads to some real weirdness.

I'm assuming you did this because you were running into spawning issues? Ideally, these functional changes should be done in a commit with a title clarifying why they were performed. It helps out reviewers and (much more importantly imo) helps out with blames months/years down the line.

<img width="218" alt="screen shot 2017-06-12 at 9 12 34 am" src="https://user-images.githubusercontent.com/714017/27035432-5244cf44-4f4f-11e7-91c3-ad0938b4387b.png">